packages/near-membrane-base/src/membrane.ts

```diff
@@ -611,6 +644,44 @@ export function createMembraneMarshall() {
             return fn;
         }
 
+        function isLiveObject(value: any): boolean {
```

Comment:
We need to figure a way for this to be called for valid proxy targets that are non-arrays, so we can avoid all the conditions below.

Reply:
Worked with @dgitin to make the checks less heavy.